words/objects.txt

```diff
@@ -1240,9 +1379,13 @@ nape
 napkin
 narwhal
 nation
+nebula
 need
 needle
+neighbor
+neighborhood
 neon
+nephew
```

Comment:
Is niece on here already?

Reply:
sure is, some weird omissions on family members